### PR TITLE
GFM-110 - removed step labels from templates

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: acceptance_tests/features/step_definitions/shared_steps.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@
 end
 
 Then(/^I can see the questions for the (.*) page of the form$/) do | header |
-  expect(page).to have_content 'Step 1'
   expect(page).to have_content CONTENT["#{header}_header"]
 end
```
Comment (JoeChapman) [inline acceptance_tests/features/step_definitions/shared_steps.rb]: Is this the only test for steps?

Comment (joefitter) [inline acceptance_tests/features/step_definitions/shared_steps.rb]: Must be if they are passing ;)

Comment (JoeChapman) [inline acceptance_tests/features/step_definitions/shared_steps.rb]: so true

**Comment by JoeChapman** [comment]
LGTM

**Pull request merged**